test/e2e/app-dir/metadata/metadata.test.ts

```diff
@@ -586,7 +586,7 @@ createNextDescribe(
 
         const browser = await next.browser('/async/not-found')
         expect(await browser.elementByCss('h2').text()).toBe(
-          'root not found page'
```

Comment:
this was wrong before, it should hit the nearby not-found boundary instead of the root one